app/views/preTaskList/CheckYourAnswersView.scala.html

```diff
@@ -37,10 +37,16 @@
 
     @heading(messages("preTaskList.checkYourAnswers.heading"))
 
+    <div class="govuk-warning-text">
```

Comment:
@kenneth-rayner sorry to be a pain - can we use the play-frontend-hmrc component? Ensures we stay in line with any future updates to this markup

https://design-system.service.gov.uk/components/warning-text/

```
@import uk.gov.hmrc.govukfrontend.views.html.components._

@this(govukWarningText : GovukWarningText)

@()(implicit messages: Messages, request: RequestHeader)


@govukWarningText(WarningText(
    iconFallbackText = Some("Warning"),
    content = Text("You can be fined up to £5,000 if you do not register.")
  ))
```